by-language/php-pdo/.gitignore

```
@@ -0,0 +1 @@
+/vendor
```

Is this correct? shouldn't it be:
```suggestion
./vendor
```

---

I am not an expert, but I think the patterns are anchored from the point where it is located.

---

It's minor really, just checked: https://git-scm.com/docs/gitignore
so:
```
For example, a pattern doc/frotz/ matches doc/frotz directory, but not a/doc/frotz directory; however frotz/ matches frotz and a/frotz that is a directory (all paths are relative from the .gitignore file).
```

So without anything, just `vendor` it's already relative to `.gitignore` dir.
I just questioned it, as I've never seen a leading `/` so far in a `.gitignore`.

---

Let's consider a typical PHP project as an example, which has this layout:

```
/vendor
/composer.lock
```

When I absolutely want to reflect that within a `.gitignore` file, I absolutely write that down there, no matter where the directory is located within a larger directory. It precisely means what is intending to describe. When, for example, just writing `vendor`, it will ignore all `vendor` folders within all other subdirectories.

Sometimes it is intended, sometimes not. For example, now looking at typical ignores of a Python project, we surely want to ignore `__pycache__` directories everywhere, so well, we just write `__pycache__`, without any `/` characters.